Gh-465: Update gaffer gremlin docs for 2.2 (#466)
gchq/gaffer-doc: docs/administration-guide/gaffer-deployment/gremlin.md

    --> D(Existing Gaffer Instance)
```

To establish this connection you can make use of the existing `gaffer-gremlin`
OCI image which is an extension of the existing `gremlin-server` image. This

> provides the Tinkerpop library to enable connecting to Gaffer graphs as well as

Reviewer: ```suggestion
provides the Tinkerpop library to allows users to connect Gaffer graphs as well as
```
This bit of the sentence reads a bit weirdly so could do with a re-phrase. I think I get what you mean but it is a bit unclear.

Reviewer 2: Shouldn't this be `Tinkerpop library to allow users to` or `Tinkerpop library which allows users to` instead of `Tinkerpop library to allows users to`?

> some pre installed configuration to get up and running quicker.

Reviewer 2: ```suggestion
some pre installed configuration to get up and running quickly.
```

```bash
docker pull gchq/gaffer-gremlin:latest
```

!!! note
    You will likely need to configure the default `gaffer-gremlin` image to your

>     environment, please continue the guide to learn more.

Reviewer 2: ```suggestion
    environment, please continue reading to learn more.
```

### Configuring the `gaffer-gremlin` Image

To use the image you will need to provide two configuration files that are specific
to your environment, they are:

> - `store.properties` - The Gaffer store configuration..
> - `gafferpop.properties` - The configuration for the Gaffer Tinkerpop library (Gafferpop).

Reviewer 2: ```suggestion
- `store.properties` - Gaffer store configuration.
- `gafferpop.properties` - Configuration for the Gaffer Tinkerpop library (Gafferpop).
```

> Once configured you can bind mount these files into the image when running the image like:

Reviewer 2: ```suggestion
Once these files are configured you can use bind mounts to make them available when running the image:
```
